app/src/main/AndroidManifest.xml

```diff
@@ -37,6 +37,13 @@
                 <data
                     android:scheme="https"
                     android:host="@string/FRONTEND_HOST"/>
+                <data
+                    android:pathPrefix="/e"/>
+                <data
+                    android:pathPrefix="/reset-password"/>
+                <data
+                    android:pathPrefix="/email"/>
+
```

Comment:
Do not commit this file at all.

Reply:
Actually before sending PR I pulled all the changes from upstream , So thats why it got added up 
What should i do Now ?